code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -457,10 +435,10 @@ emp_act
 
 /mob/living/carbon/human/blob_act()
 	if(flags & INVULNERABLE)
-		return
+		return FALSE
```

Comment:
I believe `blob_act` isn't supposed to return anything. There's no harm in doing this but why bother?

Reply:
I don't like returning null, because I have a feeling it broke something when I was dealing with #16865, as in, a var was affected to a function which returned null, and it simply stopped executing the rest of the proc (including skipping hit sounds).

Reply:
Are you sure it wasn't runtiming?

Reply:
If it did runtime, it did not print any message on the debugger.

Reply:
Well, of course, the code under that if statement wouldn't run, null is falsy. Nothing seems to expect `blob_act` to return anything, nothing assigns the value returned from it to anything, so this doesn't matter here.

Reply:
Alright